tests/suite_test.go

```diff
@@ -46,13 +47,36 @@ var (
 	PVCClient      *pvc.Kubeclient
 	DeployClient   *deploy.Kubeclient
 	PodClient      *pod.KubeClient
+	ZVClient       *zv.Kubeclient
 	scName         = "zfspv-sc"
 	ZFSProvisioner = "zfs.csi.openebs.io"
-	pvcName        = "zfspv-pvc"
-	snapName       = "zfspv-snap"
-	appName        = "busybox-zfspv"
-	clonePvcName   = "zfspv-pvc-clone"
-	cloneAppName   = "busybox-zfspv-clone"
+
+	pvcName  = "zfspv-pvc-fs"
+	snapName = "zfspv-snap"
+	appName  = "busybox-zfspv"
+
+	clonePvcName = "zfspv-pvc-clone"
+	cloneAppName = "busybox-zfspv-clone"
+
+	pvcNameFS    = "zfspv-pvc-fs"
+	pvcNameBlock = "zfspv-pvc-block"
+	pvcNameAplha = "zfspv-pvc-alpha"
+
+	appNameFS    = "busybox-zfspv-fs"
+	appNameBlock = "busybox-zfspv-block"
+	appNameAlpha = "busybox-zfspv-alpha"
```

Comment:
what's alpha?

Reply:
Why does it need to be different? Shouldn't blockVolume tests using the blockVol pvc naming?

Reply:
since the test was running in parallel (on same cluster)  there were name conflicts. Thats why i used alpha.  reverted it now.